sled-agent/src/opte/opte.rs

```diff
@@ -320,6 +320,7 @@ impl Drop for OptePort {
 
 /// Delete all xde devices on the system.
 pub fn delete_all_xde_devices(log: &Logger) -> Result<(), Error> {
+    // Delete the VNICs as well.
```

Comment:
Is the deletion of VNICs implied by `delete_xde`?  I'm trying to understand why this comment is here.

Reply:
No, it's not. The deletion of the VNICs is a separate step. That's a stray comment, will remove.